test/question.spec.js

```
@@ -30,6 +32,8 @@ describe('Question Api Exists', () => {
         .send(params)
         .end((err, res) => {
           expect(res.statusCode).to.equal(201);
+          expect(res.body).to.be.an('object');
+          expect(res.body).to.have.property('data');
```

Why are you only checking for the property data?

---

I've been having issues working my way around checking for the keys in my data object. I'm thinking the error is because I am using a dummy database because when I use `expect(res.body).to.have.keys({ 'title', 'body' })`, my test kept failing saying it's expecting an additional key '0'. So I decided to only check for the variable holding the object inside.